tensorboard/webapp/widgets/card_fob/card_fob_component.scss

```diff
@@ -19,20 +19,43 @@ limitations under the License.
   display: inline-block;
 }
 
+$FOB_BACKGROUND: mat.get-color-from-palette(mat.$gray-palette, 300);
+
 .fob {
   display: inline-flex;
-  background-color: mat.get-color-from-palette(mat.$gray-palette, 300);
+  background-color: $FOB_BACKGROUND;
```

Comment:
why make it an variable while it's only used once?

Reply:
I'll revert this.